README.md

```diff
@@ -39,7 +40,7 @@ Ordnance Survey is the national mapping agency for Great Britain and produces a
 
 - [x] OS Names API
 - [x] OS Places API
-- [ ] OS NGD Features API (Coming Soon!)
+- [x] OS NGD Features API (Coming Soon!)
```

Comment:
Remove the checkboxes and the comming soon

Reply:
Removed